card/src/main/java/com/adyen/checkout/card/internal/ui/model/InstallmentsParamsMapper.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 22/2/2023.
+ */
+
+package com.adyen.checkout.card.internal.ui.model
+
+import com.adyen.checkout.card.CardBrand
+import com.adyen.checkout.card.InstallmentConfiguration
+import com.adyen.checkout.card.InstallmentOptions
+import com.adyen.checkout.sessions.core.SessionSetupInstallmentOptions
+
+internal class InstallmentsParamsMapper {
+
+    internal fun mapToInstallmentParams(
+        sessionInstallmentOptions: Map<String, SessionSetupInstallmentOptions?>?,
+        installmentConfiguration: InstallmentConfiguration?
+    ): InstallmentParams? {
+        return if (sessionInstallmentOptions != null) {
+            var defaultOptions: InstallmentOptionParams.DefaultInstallmentOptions? = null
```

Comment:
Minor: let's extract this logic to a method as well? This way it reads a bit nicer:
```kotlin
return sessionInstallmentOptions?.mapToInstallmentParams()
? installmentConfiguration?.mapToInstallmentParams()
```

WDYT?